<launch>
    <rosparam file="$(find xiaohu_robot)/manipulator/params/simulation.yaml" command="load" />
    <node
        name="controller_spawner"
        pkg="controller_manager"
        type="spawner"
        respawn="false"
        output="log"
        ns="/wpb_home"
        args="
            joint_state_controller
            mani_base_position_controller
            elbow_forearm_position_controller
            palm_left_finger_position_controller
            left_finger_tip_position_controller
            palm_right_finger_position_controller
            right_finger_tip_position_controller
        "
    />
    <param name="wpb_home_description" textfile="$(find xiaohu_robot)/gazebo/models/robot.model" />
    <node name="gazebo_wpb_home_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
        <remap from="robot_description" to="wpb_home_description" />
        <remap from="/joint_states" to="/wpb_home/joint_states" />
    </node>
    <node name="core_sim_node" pkg="xiaohu_robot" type="core_sim_node" />
</launch>